learning_service/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7.10-slim
+
+RUN apt-get update \
+&& apt-get install -y --no-install-recommends git \
+&& apt-get purge -y --auto-remove \
+&& rm -rf /var/lib/apt/lists/*
+
+WORKDIR /root/
+
+ENV VIRTUAL_ENV=/root/venv
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+COPY requirements.txt .
+RUN python -m pip install --upgrade pip &&\
+    pip install -r requirements.txt
+
+EXPOSE 8080
+
+CMD "bash"
```

Comment:
Shouldn't `CMD` run the application server?

Reply:
We do not have an application yet. We still need to figure out handling queue events on new data and such.

Reply:
Oh true, we gotta implement the blocking thread for handling the pubsub data.